backend/src/graphql/resolvers-types.ts

```diff
@@ -2,12 +2,9 @@ import { GraphQLResolveInfo, GraphQLScalarType, GraphQLScalarTypeConfig } from '
 import { ApolloContext } from '../server/apolloContext';
 export type Maybe<T> = T | null;
 export type Exact<T extends { [key: string]: unknown }> = { [K in keyof T]: T[K] };
-export type MakeOptional<T, K extends keyof T> = Omit<T, K> &
```

Comment:
This change is not related to this merge request. 

(Why there is formatting change? 🤔  `./backend/codegen.yml` should run the prettier automatically after each update)

Reply:
Yeah, I've done too many tests on this branch. I won't merge it. I'll just copy the final action file to a new branch :)